The sqlite configuration is assuming that compiler with no -mXX flag compiles
32 bit binaries. This is no longer true, as the default is 64bit these days.
This patch fixes the 32bit compilation.

This patch has been reported to sqlite-users@mailinglists.sqlite.org

http://mailinglists.sqlite.org/cgi-bin/mailman/private/sqlite-users/2017-November/076381.html

--- sqlite-autoconf-3400100/tea/tclconfig/tcl.m4
+++ sqlite-autoconf-3400100/tea/tclconfig/tcl.m4
@@ -1871,7 +1871,7 @@ AC_DEFUN([TEA_CONFIG_CFLAGS], [
 			# not be necessary for extensions.
 			SHLIB_LD="$SHLIB_LD -m64 -static-libgcc"
 		    ])])
-		])
+		], [ SHLIB_LD="$SHLIB_LD -m32" ])
 	    ], [
 		case $system in
 		    SunOS-5.[[1-9]][[0-9]]*)
